src/components/MenuItem.js

```diff
@@ -137,6 +142,35 @@ const MenuItem = React.forwardRef((props, ref) => {
 
     const hasPressableRightComponent = props.iconRight || (props.rightComponent && props.shouldShowRightComponent);
 
+    const renderTitleContent = () => {
+        const isEllipsisActive = lodashGet(titleContainerRef.current, 'offsetWidth') < lodashGet(titleContainerRef.current, 'scrollWidth');
+
+        if (props.titleWithTooltips && _.isArray(props.titleWithTooltips) && props.titleWithTooltips.length > 0) {
+            return (
+                <Text>
+                    {_.map(props.titleWithTooltips, (tooltipDetails, index) => (
+                        <Text key={index}>
+                            <UserDetailsTooltip accountID={tooltipDetails.accountID}>
+                                <Text>{convertToLTR(tooltipDetails.displayName)}</Text>
+                            </UserDetailsTooltip>
+                            {index < props.titleWithTooltips.length - 1 && <Text style>,&nbsp;</Text>}
+                        </Text>
+                    ))}
+                    {Boolean(isEllipsisActive) && (
+                        <View style={styles.displayNameTooltipEllipsis}>
+                            <Tooltip text={props.title}>
+                                {/* There is some Gap for real ellipsis so we are adding 5 `.` to cover */}
```

Comment:
Can you explain why this is needed?

Reply:
@Krishna2323 Oh wait I see you've taken this from `DisplayNamesWithTooltip`. I'm wondering now if there's any reason why we don't just use `DisplayNames` here actually, rather than reproducing most of the code from it?